x/tokenomics/module/abci.go

```diff
@@ -10,29 +10,49 @@ import (
 )
 
 // EndBlocker called at every block and settles all pending claims.
-func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) (err error) {
 	logger := k.Logger().With("method", "EndBlocker")
 	// NB: There are two main reasons why we settle expiring claims in the end
 	// instead of when a proof is submitted:
 	// 1. Logic - Probabilistic proof allows claims to be settled (i.e. rewarded)
 	//    even without a proof to be able to scale to unbounded Claims & Proofs.
 	// 2. Implementation - This cannot be done from the `x/proof` module because
 	//    it would create a circular dependency.
-	numClaimsSettled, numClaimsExpired, relaysPerServiceMap, err := k.SettlePendingClaims(ctx)
+	numClaimsSettled,
```

Comment:
This TODO already exists above the `#SettlePendingClaim()` definition:

```
// TODO_TECHDEBT: Refactor this function to return a struct instead of multiple return values.
```